bundles/org.openhab.binding.luxom/src/main/java/org/openhab/binding/luxom/internal/handler/LuxomBridgeHandler.java

```diff
@@ -0,0 +1,354 @@
+import org.openhab.binding.luxom.internal.protocol.LuxomCommunication;
+import org.openhab.binding.luxom.internal.protocol.LuxomSystemInfo;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handler responsible for communicating with the main Luxom IP access module.
+ *
+ * @author Kris Jespers - Initial contribution
+ */
+@NonNullByDefault
+public class LuxomBridgeHandler extends BaseBridgeHandler {
+    public static final int HEARTBEAT_INTERVAL_SECONDS = 50;
+    private final LuxomSystemInfo systemInfo;
+
+    private static final int DEFAULT_RECONNECT_INTERVAL_IN_MINUTES = 1;
+    private static final long HEARTBEAT_ACK_TIMEOUT_SECONDS = 20;
+
+    private final Logger logger = LoggerFactory.getLogger(LuxomBridgeHandler.class);
+    private final LocalizationService localizationService;
+
+    private @Nullable LuxomBridgeConfig config;
+    private final AtomicInteger nrOfSendPermits = new AtomicInteger(0);
+    private int reconnectInterval;
+
+    private @Nullable LuxomCommand previousCommand = null;
+    private final LuxomCommunication communication;
+    private final BlockingQueue<List<CommandExecutionSpecification>> sendQueue = new LinkedBlockingQueue<>();
+
+    private @Nullable Thread messageSender;
+    private @Nullable ScheduledFuture<?> heartBeat;
+    private @Nullable ScheduledFuture<?> heartBeatTimeoutTask;
+    private @Nullable ScheduledFuture<?> connectRetryJob;
+
+    @Nullable
+    public LuxomBridgeConfig getIPBridgeConfig() {
+        return config;
+    }
+
+    public LuxomBridgeHandler(Bridge bridge, LocalizationService localizationService) {
+        super(bridge);
+        logger.info("{}", localizationService.getText("status.bridge-initializing", "Luxom bridge init"));
+        this.localizationService = localizationService;
+        this.systemInfo = new LuxomSystemInfo();
+        this.communication = new LuxomCommunication(this);
+    }
+
+    @Override
+    public void handleCommand(@NotNull ChannelUID channelUID, Command command) {
+        logger.debug("bridge received command {} for {}", command.toFullString(), channelUID);
+    }
+
+    @Override
+    public void initialize() {
+        this.config = getThing().getConfiguration().as(LuxomBridgeConfig.class);
+
+        if (validConfiguration(this.config)) {
+            reconnectInterval = (config.reconnectInterval > 0) ? config.reconnectInterval
+                    : DEFAULT_RECONNECT_INTERVAL_IN_MINUTES;
+
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.NONE,
+                    localizationService.getText("status.connecting", "Connecting"));
```

Comment:
You can simply use the string "@text/status.connecting":
```suggestion
                    "@text/status.connecting");
```